docs/source/actor-rhel7-o-rhel8.md

```diff
@@ -0,0 +1,315 @@
+# How to create a Leapp actor for RHEL 7 to 8 upgrade
+
+## Introduction
+
+This document is intended for all people who want to contribute to RHEL upgrade process using Leapp tool. After reading through this document, you will be able to transform your expertise in certain parts of RHEL into improvements of the RHEL 7 to 8 upgrade tooling. You are expected to be familiar with the concept of in-place upgrades and ideally to have at least high-level knowledge about the Leapp tool.
+
+## Setting up the development environment
+
+Leapp actors are written in Python, so your usual Python development setup can be used during the process of creating a new actor.
```

Comment:
This may raise a question which exactly python. Should we specify that code has to be compatible with both py2 and py3?